package.json

```diff
@@ -5,18 +5,17 @@
   "license": "MIT",
   "scripts": {
     "start": "react-scripts start",
+    "install-start": "npm install && npm run lintnofix && npm run start",
     "build": "npm install && npm run lintnofix && npm run rsbuild",
     "rsbuild": "react-scripts build",
     "snapshot": "react-snap",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject",
```

Comment:
I would keep this since it's a standard part of CRA.  I have no plans to eject, but we should leave it there

Reply:
I don't want people to accidentally run it. It's not reversible. We should keep it hidden (since it will always be there as part of react-scripts)